archive/terraform/kms_key/encryption_kms_key.tf

```diff
@@ -0,0 +1,4 @@
+resource "aws_kms_key" "encryption_key" {
```

Comment:
We generally don't bother with KMS, but obviously this is better than not. Probably should spread the pattern everywhere.